mobile/package.json

```diff
@@ -42,7 +44,8 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo web",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "dev": "export BASE_URL=http://localhost:8080 && npm start"
```

Comment:
This should just be `BASE_URL=http://localhost:8080 npm start`, no export or &&.